live-examples/js-examples/date/date-toprimitive.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">
+  const date = new Date('20 December 2019 14:48'); //initialize a new Date object
+
+  console.log(date[Symbol.toPrimitive]('string')); //log primitive of date with hint as string
+  //expected output: "Fri Dec 20 2019 20:18:00 GMT+0530 (India Standard Time)"
+  //note: the timezone specific data(GMT+0530 (India Standard Time)) may vary according to your location
+
+  console.log(date[Symbol.toPrimitive]('number')); //log primitive of date with hint as number
+  //expected output: 1576853280000 ,this may also vary
```

Comment:
rather than ` ,this may also vary` it would be better to have this on a separate line and restate the issue:

`// timezone specific data may vary according to your local settings`